test/utils-tests.jl

```diff
@@ -346,6 +346,30 @@ end
     @test hlat_out[1] ≈ -0.6845757329085267
 end
 
+# Test hor2eq
+# The values used for the testset are from running the code. However they have been
+# correlated with the output from hor2eq routine of IDL AstroLib, with
+# differences only in the least significant digits.
+@testset "hor2eq" begin
+    ra_o, dec_o, ha_o = hor2eq(43.6879, 56.684, AstroLib.J2000, B1950=true)
+    @test ra_o ≈ 259.20005705918317
+    @test dec_o ≈ 49.674706171288655
+    @test ha_o ≈ 19.40545272279752
```

Comment:
I think the difference stems from the use of the updated values in  `true_obliquity` and `mean_obliquity`; in `co_aberration` and `co_nutate` respectively ( #24 )